Core/clim-core/dialog.lisp

```diff
@@ -440,20 +440,24 @@ highlighting, etc." ))
   (apply #'prompt-for-accept-1 stream type :display-default nil args))
 
 (define-command (com-query-exit :command-table accept-values
-                                :keystroke (#\] :control)
+                                :keystroke (#\[ :control)
                                 :name nil
                                 :provide-output-destination-keyword nil)
     ()
   (signal 'av-exit))
 
+(add-keystroke-to-command-table 'accept-values '(#\Return :meta) :command #'com-query-exit)
```

Comment:
Have You tried running and testing this? I think `:command` needs to be a symbol so `'com-query-exit` will work but `#'com-query-exit` wouldn't.